### ISSUE TITLE
Getting requests errors

### ISSUE THREAD

**Issue by Chokitu** [tweepy/tweepy#1519]
So, I was getting 429 errors ever since yesterday, I had wait on rate limit as true, and notify as well. I had time sleep and not too many items being searched.

Even after 15 hours, I was still not able to run the bot, because of 429 error. 

Today I woke up to this: {'message': 'Request exceeds account’s current package request limits.  {'message': 'Request exceeds account’s current packages

I changed the code a bit, changed the full_archive_search to a regular query search and got the 429 errors again.

In the developers dashboard it shows that I have 0 requests pulled, and my bot only got like 26 tweets favorited.

Does anyone know why?

**Comment by joseph-allen** [comment]
Same issue here, same error message.

Strange as I have the academic tier too, but can't seem to see any of the tickers increasing on the Twitter Developer Portal.

**Comment by jalalshabo** [comment]
Some issue, streaming is working for fine but using the timeline tweets I get a not autherized error when the code was working perfectly previously

**Comment by joseph-allen** [comment]
I found that while I hadn't hit the tweet limit, I had hit the number of requests I was allowed per month. It was not intuitive to find on the platform though.

Check here -> https://developer.twitter.com/en/dashboard

**Comment by edsmon** [comment]
> I found that while I hadn't hit the tweet limit, I had hit the number of requests I was allowed per month. It was not intuitive to find on the platform though.
> 
> Check here -> https://developer.twitter.com/en/dashboard

Thanks @joseph-allen

**Comment by Harmon758** [comment]
Without the full traceback and/or relevant code, this isn't actionable, but this was likely resolved with https://github.com/tweepy/tweepy/commit/cd5f696d09530f86ac0edf1ec0fe0a02578a3920 in [Tweepy v4.0.0](https://github.com/tweepy/tweepy/releases/tag/v4.0.0).

Feel free to open a new issue with the full traceback and/or relevant code if you can still reproduce this with the latest version of Tweepy, currently v4.2.0.

As for why this is happening, it's likely that you're hitting your limit for requests per month for the full-archive search endpoint.
See the [premium search API documentation](https://developer.twitter.com/en/docs/twitter-api/premium/search-api/overview) for more information.
